partner_autocreate_task/models/partner_task_autocreate.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# © 2017 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
+
+
+_intervalTypes = {
+    'work_days': lambda interval: relativedelta(days=interval),
+    'days': lambda interval: relativedelta(days=interval),
+    'hours': lambda interval: relativedelta(hours=interval),
+    'weeks': lambda interval: relativedelta(days=7*interval),
+    'months': lambda interval: relativedelta(months=interval),
+    'years': lambda interval: relativedelta(years=interval),
+}
+
+
+class PartnerTaskAutoCreate(models.Model):
+    """Partner Task Auto Create"""
+
+    _name = 'partner.task.autocreate'
+    _description = __doc__
+    _rec_name = 'task_id'
+
+    @api.depends('last_created', 'interval_number', 'interval_type')
+    def _compute_nextcall(self):
+        """
+        Compute the nextcall value.
+        """
+        for rec in self:
+            next_call = fields.Datetime.now()
+            if rec.last_created:
+                last = fields.Datetime.from_string(rec.last_created)
+                next_call = last + (
+                    _intervalTypes[rec.interval_type](rec.interval_number)
+                )
+            rec.update({
+                'nextcall': next_call
+            })
+
+    task_id = fields.Many2one(
```

Comment:
@rimbendhaou project.task.template comes from the module `project_task_autocreate` which is not in the dependecies list.

Reply:
@joaoalf ok